docker-build.sh

```diff
@@ -5,6 +5,123 @@
 set -o errexit
 set -o xtrace
 
+ARCHIVE_ADDR=https://archive.ubuntu.com/ubuntu/
+PORTS_ADDR=https://ports.ubuntu.com/
+
+# Prepare HWA headers, libs and drivers for x86_64-linux-gnu
+prepare_hwa_amd64() {
+    # Download and install the nvidia headers from deb-multimedia
+    pushd ${SOURCE_DIR}
+    git clone --depth=1 https://git.videolan.org/git/ffmpeg/nv-codec-headers.git
+    pushd nv-codec-headers
+    make
+    make install
+    popd
+    popd
+
+    # Download and setup AMD AMF headers from AMD official github repo
+    # https://www.ffmpeg.org/general.html#AMD-AMF_002fVCE
+    svn checkout https://github.com/GPUOpen-LibrariesAndSDKs/AMF/trunk/amf/public/include
+    pushd include
+    mkdir -p /usr/include/AMF && mv * /usr/include/AMF
+    popd
+
+    # Download and install libva
+    pushd ${SOURCE_DIR}
+    git clone -b v2.6-branch --depth=1 https://github.com/intel/libva
+    pushd libva
+    sed -i 's|getenv("LIBVA_DRIVERS_PATH")|"/usr/lib/jellyfin-ffmpeg/dri"|g' va/va.c
+    sed -i 's|getenv("LIBVA_DRIVER_NAME")|NULL|g' va/va.c
+    ./autogen.sh
+    ./configure --prefix=/usr
+    make -j$(nproc) && make install
+    ./configure --libdir=${SOURCE_DIR}/intel-drivers
+    make install
+    echo "intel-drivers/libva.so* usr/lib/jellyfin-ffmpeg/libs" >> ${SOURCE_DIR}/debian/jellyfin-ffmpeg.install
+    echo "intel-drivers/libva-drm.so* usr/lib/jellyfin-ffmpeg/libs" >> ${SOURCE_DIR}/debian/jellyfin-ffmpeg.install
+    popd
+    popd
+
+    # Download and install intel-vaapi-driver
+    pushd ${SOURCE_DIR}
+    git clone -b v2.4-branch --depth=1 https://github.com/intel/intel-vaapi-driver
+    pushd intel-vaapi-driver
+    ./autogen.sh
+    ./configure --prefix=/usr
```

Comment:
same about `prefix`, it should point to the correct location from the get-go, not relying on the fact that it works _for now_ when we move files